net.c

```diff
@@ -143,7 +143,6 @@ int redisKeepAlive(redisContext *c, int interval) {
     }
 #else
 #if defined(__GLIBC__) && !defined(__FreeBSD_kernel__)
-    val = interval;
```

Comment:
Because it is already set before
```cpp
     val = interval;
 		 
 #ifdef _OSX		 #ifdef _OSX
     if (setsockopt(fd, IPPROTO_TCP, TCP_KEEPALIVE, &val, sizeof(val)) < 0) {		     if (setsockopt(fd, IPPROTO_TCP, TCP_KEEPALIVE, &val, sizeof(val)) < 0) {
         __redisSetError(c,REDIS_ERR_OTHER,strerror(errno));		         __redisSetError(c,REDIS_ERR_OTHER,strerror(errno));
         return REDIS_ERR;		         return REDIS_ERR;
      }		      }
  #else		  #else
  #if defined(__GLIBC__) && !defined(__FreeBSD_kernel__)		  #if defined(__GLIBC__) && !defined(__FreeBSD_kernel__)
 -    val = interval;		
```